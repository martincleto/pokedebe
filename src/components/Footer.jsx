
import React, { Component } from 'react'

import styles from 'Stylesheets/footer.scss'  // eslint-disable-line no-unused-vars

class Footer extends Component {
  constructor(props) {
    super(props)
  }

  render() {
    return (
      <footer className="footer">
        <p>Made with <span className="icon-love">love</span> by <a href="http://github.com/martincleto" title="martincleto at GitHub">martincleto</a> to Ainara & Enric</p>
      </footer>
    )
  }
}

export default Footer
