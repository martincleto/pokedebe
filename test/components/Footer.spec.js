import React from 'react'
import renderer from 'react-test-renderer'
import Footer from 'Components/Footer'

describe('Footer', function() {
  test('should render without error', () => {
    const component = renderer.create(
      <Footer/>
    ).toJSON();

    expect(component).toMatchSnapshot();
  });
})
