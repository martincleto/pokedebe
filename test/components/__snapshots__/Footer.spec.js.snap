// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render without error 1`] = `
<footer
  className="footer"
>
  <p>
    Made with 
    <span
      className="icon-love"
    >
      love
    </span>
     by 
    <a
      href="http://github.com/martincleto"
      title="martincleto at GitHub"
    >
      martincleto
    </a>
     to Ainara & Enric
  </p>
</footer>
`;
